tornado_elasticsearch.py

```diff
@@ -324,6 +326,30 @@ def index(self, index, doc_type, body, id=None, params=None):
         raise gen.Return(data)
 
     @gen.coroutine
+    @query_params('allow_no_indices', 'expand_wildcards', 'ignore_unavailable',
+        'local')
+    def exists_index(self, index, params=None):
```

Comment:
Can you move this to a submodule called indices? Easier option -> leave a comment as reminder that this function needs to be accessible as `indices.exists`, not `exists_index`.